source/_data/glossary.yml

```diff
@@ -239,6 +239,18 @@
     Home Assistant OS, the Home Assistant Operating System, is an embedded,
     minimalistic, operating system designed to run the Home Assistant ecosystem.
 
+- term: Host
+  definition: >-
+    A device that can communicate with other devices on a network. During setup and configuration,
+    an input requesting a *Host* typically refers to a device's network address so that
+    Home Assistant can attempt to connect to it. This may be in the form of a host name, URL,
+    IP address or some other type of network identifier.
```

Comment:
```suggestion
    A device that can communicate with other devices on a network. During setup and configuration,
    an input requesting a **Host(* typically refers to a device's network address so that
    Home Assistant can attempt to connect to it. This may be in the form of a hostname, URL,
    IP address or some other type of network identifier.
```